Frameworks/Foundation/NSUndoManager.mm

```diff
@@ -14,18 +14,84 @@
 //
 //******************************************************************************
 
+#import <Starboard.h>
 #import <StubReturn.h>
 #import <Foundation/NSUndoManager.h>
+#import "_NSUndoManagerInternal.h"
 
 NSString* const NSUndoManagerGroupIsDiscardableKey = @"NSUndoManagerGroupIsDiscardableKey";
 
-@implementation NSUndoManager
+@implementation NSUndoManager {
+    StrongId<_NSUndoManagerStack> _undoStack;
+    StrongId<_NSUndoManagerStack> _redoStack;
+    NSUInteger _undoGroupingLevel;
+    NSUInteger _redoGroupingLevel;
+    NSUInteger _disableCount;
+    BOOL _isAutomaticGroupOpen;
+    BOOL _hasNestedGroup;
+}
+
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
+*/
+- (id)init {
+    if (self = [super init]) {
+        _runLoopModes = @[ NSDefaultRunLoopMode ];
```

Comment:
how do Good Practices feel about using the setter in the initializer here to implicitly ensure that this has the right retain level?

---
In reply to: [76707964](https://github.com/Microsoft/WinObjC/pull/900#discussion_r76707964) [](ancestors = 76707964)